lib/caps/LNL/info

```diff
@@ -0,0 +1,13 @@
+###
+### Copyright (C) 2024 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+###
+### kate: syntax python;
+###
+
+info = dict(
+  gpu = dict(gen = 13),
```

Comment:
LNL is Gen 15

Reply:
Thank you for the review, it has been updated.